libvast/vast/concept/parseable/core/repeat.hpp

```diff
@@ -88,8 +85,8 @@ class dynamic_repeat_parser
 
 private:
   Parser parser_;
-  const T& min_;
-  const U& max_;
+  const T min_;
+  const T max_;
```

Comment:
Just a heads up: You can likely revert this change tomorrow, as I plan to fix this (and the same bug in the skip parser) upstream.

Reply:
This has been fixed upstream.